src/core/analog_controller.cpp

```diff
@@ -29,7 +29,10 @@ AnalogController::AnalogController(u32 index) : Controller(index)
   m_rumble_config.fill(0xFF);
 }
 
-AnalogController::~AnalogController() = default;
+AnalogController::~AnalogController()
+{
+  InputManager::SetGamepadAnalogLED(m_index, false);
```

Comment:
This is the index of the emulated controller, which does not necessarily correspond to the player index of the connected controller. Emulated controller 0 can be mapped to SDL player 1, or vice versa, etc. So really this should have a bind and get forwarded through, similar to vibration.